.changeset/sixty-teachers-tap.md

```diff
@@ -0,0 +1,26 @@
+---
+'@astrojs/vercel': major
```

Comment:
Since we're only deprecating the `analytics` option now, could this be marked as a minor instead?